syntax = "proto2";

package rv.protos;

import "rastervision/protos/machine_learning.proto";
import "rastervision/protos/scene.proto";


message EvalConfig {
    message Options {
        optional string output_uri = 1;
        optional bool debug = 4 [default=true];
    }

    required MachineLearning machine_learning = 1;
    repeated Scene scenes = 4;
    required Options options = 5;
}
